### ISSUE TITLE
cant setup the integration

### ISSUE THREAD

**Issue by Bram-diederik** [TECH7Fox/asterisk-hass-integration#85]
After a memory leak i removed the integration. 
I was going to try it out again. 
but the setup leaves a integration with no identities. 

```
023-05-25 15:58:37.505 ERROR (MainThread) [homeassistant.config_entries] Error unloading entry AMI 192.168.5.43:5038 for binary_sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 543, in async_unload
    result = await component.async_unload_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 175, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 194, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
2023-05-25 15:58:37.515 ERROR (MainThread) [homeassistant.config_entries] Error unloading entry AMI 192.168.5.43:5038 for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 543, in async_unload
    result = await component.async_unload_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 137, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 194, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
```

**Comment by mitchmitchell** [comment]
I'm seeing the same when I try to reload the integration:
Logger: homeassistant.config_entries
Source: helpers/entity_component.py:194
First occurred: 7:14:57 PM (2 occurrences)
Last logged: 7:14:57 PM

Error unloading entry AMI pbx.magnoliamanor.local:5038 for binary_sensor
Error unloading entry AMI pbx.magnoliamanor.local:5038 for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 543, in async_unload
    result = await component.async_unload_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 175, in async_unload_entry
    return await component.async_unload_entry(entry)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 194, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!


It was definitely connected to my FreePBX server but now no longer shows up in Asterism "manager show connected" output

**Comment by mitchmitchell** [comment]
This issue was introduced on may 8th commit 9e6c1e4 -- working on a pull request now.

**Comment by TECH7Fox** [comment]
Should be fixed now in v1.0.3. Please reopen if you're still having this issue.
